less/general-style.less

```diff
@@ -0,0 +1,25 @@
+.container {
+    max-width: 880px;
+}
+
+.skills {
+
+    h3 {
+        display: flex;
+        justify-content: center;

```

Comment:
Is this the reason your card keeps popping over when you don't want it to? Its inheriting justify-content from the skills div. Maybe doing something like a flex-end on it or a position fixed and put it exactly where you want?

Reply:
Thanks Nate, I was able to fix the card issue with a later submission.  I think it was the max-height I had put on the .skills class, but I could be mistaken.  Thanks for looking into it.